class.jetpack-connection-banner.php

```diff
@@ -424,13 +465,250 @@ class="dops-button is-primary">
+							</div>
+							<span class="vertical-menu__feature-item-label"><?php esc_html_e( 'Create Your Site', 'jetpack' ); ?></span>
+						</div>
+						<div class="vertical-menu__feature-item">
+							<div class="vertical-menu__feature-item-icon">
+								<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 2 20 20" version="1.1"><path d="M7.8 17.6L12.2 17.6 12.2 2 7.8 2 7.8 17.6ZM14.4 17.6L18.9 17.6 18.9 5.3 14.4 5.3 14.4 17.6ZM1.1 17.6L5.6 17.6 5.6 9.8 1.1 9.8 1.1 17.6ZM0 22L20 22 20 19.8 0 19.8 0 22Z" /></svg>
+							</div>
+							<span class="vertical-menu__feature-item-label"><?php esc_html_e( 'Make It Successful', 'jetpack' ); ?></span>
+						</div>
+						<div class="vertical-menu__feature-item">
+							<div class="vertical-menu__feature-item-icon">
+								<svg xmlns="http://www.w3.org/2000/svg" width="16" height="20" viewBox="0 1 16 20" version="1.1"><defs><polygon points="16 10 16 0 0 0 0 10 0 20 16 20"/></defs><g stroke="none" stroke-width="1" transform="translate(0.000000, 1.000000)"><mask fill="white"/><path d="M9 13.7L9 16 7 16 7 13.7C6.4 13.4 6 12.7 6 12 6 10.9 6.9 10 8 10 9.1 10 10 10.9 10 12 10 12.7 9.6 13.4 9 13.7L9 13.7ZM5 5C5 3.3 6.3 2 8 2 9.7 2 11 3.3 11 5L11 6 5 6 5 5ZM14 6L13 6 13 5C13 2.2 10.8 0 8 0 5.2 0 3 2.2 3 5L3 6 2 6C0.9 6 0 6.9 0 8L0 18C0 19.1 0.9 20 2 20L14 20C15.1 20 16 19.1 16 18L16 8C16 6.9 15.1 6 14 6L14 6Z" mask="url(#mask-2)"/></g></svg>
+							</div>
+							<span class="vertical-menu__feature-item-label"><?php esc_html_e( 'Keep It Safe', 'jetpack' ); ?></span>
+						</div>
+					</div>
+				</div>
+				<div class="jp-wpcom-connect__content-container">
+
+					<!-- slide 1: intro -->
+					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-one jp__slide-is-active">
+						<h2><?php esc_html_e( 'Welcome to Jetpack', 'jetpack' ) ?></h2>
+
+						<div class="jp-wpcom-connect__content-icon jp-connect-illo">
+							<img src="<?php echo plugins_url( 'jetpack/images/jetpack-welcome.svg', dirname( __FILE__ ) ); ?>" alt="Your site is automatically protected from brute force attacks, plus you can use single sign-on for extra security." height="auto" width="250" />
+						</div>
+
+						<p>
+							<?php
+							esc_html_e(
+								'Jetpack is the best way to experience WordPress, whether your site is brand new or already well established.',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p>
+							<?php
+							esc_html_e(
+								'You get themes and tools to design your site, marketing services to make it successful, and state-of-the-art security.',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p>
+							<?php
+							esc_html_e(
+								'Connect to WordPress.com (free) to get started.',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p class="jp-banner__button-container">
+							<a
+								href="<?php echo esc_url( Jetpack::init()->build_connect_url( true, false, 'banner-53-slide-1' ) ); ?>"
```

Comment:
In `render_banner()`, we're appending `$current_screen->base` to the tracks string so we can determine which page it was clicked on (It's now displaying on both the dashboard and plugins page now).  Let's add that in this banner as well.